hack/kubestellar.sh

```diff
@@ -0,0 +1,219 @@
+then
+    echo "An older deployment of mailbox-controller is already running - terminating it ...."
+    pkill -f mailbox-controller
+fi
+
+# Check edge-scheduler is already running
+if [ $(process_running "scheduler -v 2") == "running" ]
+then
+    echo "An older deployment of edge-scheduler is already running - terminating it ...."
+    pkill -f "scheduler -v 2"
+fi
+
+# Check placement-translator is already running
+if [ $(process_running placement-translator) == "running" ]
+then
+    echo "An older deployment of placement-translator is already running - terminating it ...."
+    pkill -f placement-translator
+fi
+
+kubectl ws root &> /dev/null
+
+if [ $install -eq 0 ]; then 
+   if kubectl get Workspace "$espw_name" &> /dev/null; then
+      kubectl delete ws "$espw_name"
+      echo "Deleted workspace: $espw_name"
+   fi
+   echo "kcp edge stopped ....."
+   exit
+fi
+
+
+wait_for_process(){
+  status=$(process_running $1)
+  MAX_RETRIES=5
+  retries=0
+  status_code=0
+  while [ $status != "running" ]; do
+      if [ $retries -eq $MAX_RETRIES ]; then
+           status_code=1
+           break
+      fi
+
+      retries=$(( retries + 1 ))
+      sleep 3
+      status=$(process_running $1)
+  done
+  echo $status_code
+}
+
+
+# Start the edge-mc controllers
+echo "****************************************"
+echo "Launching KubeStellar ..."
+echo "****************************************"
+
+
+if kubectl get Workspace "$espw_name" &> /dev/null; then
+   echo "espw workspace already exists -- using it:"
+   kubectl ws "$espw_name"
+else 
+   if [ $verbosity == 1 ]; then
+        kubectl ws create "$espw_name" --enter
+        kubectl apply -f  ./config/crds 
+        kubectl apply -f  ./config/exports
+        echo "Finished populate the espw with kcp edge crds and apiexports"
+   else
+        kubectl ws create "$espw_name" --enter
+        kubectl apply -f  ./config/crds &> /dev/null
+        kubectl apply -f  ./config/exports &> /dev/null
+        echo "Finished populate the espw with kcp edge crds and apiexports"
+   fi
+fi
+
+sleep 5
+
+# Create the logs directory
+if [[ ! -d $log_folder ]]; then
+    mkdir -p $log_folder
+fi
+
+mailbox-controller -v=2 >& $log_folder/mailbox-controller-log.txt &
+
+run_status=$(wait_for_process mailbox-controller)
+if [ $run_status -eq 0 ]; then
+    echo " mailbox-controller is running (log file: mailbox-controller-log.txt)"
+else
+    echo " mailbox-controller failed to start ..... exiting"
+    sleep 2
+    exit
+fi
+
+
+# Start the edge-mc scheduler
+sleep 3
+scheduler -v 2 >& $log_folder/edge-scheduler-log.txt &
+
+run_status=$(wait_for_process "scheduler -v 2")
+if [ $run_status -eq 0 ]; then
+    echo " scheduler is running (log file: edge-scheduler-log.txt)"
+else
+    echo " scheduler failed to start ..... exiting"
+    exit
+fi
+ 
+# Start the Placement Translator
+sleep 3
+placement-translator --allclusters-context  "system:admin" -v=2 >& $log_folder/placement-translator-log.txt &
```

Comment:
The `--allclusters-context  "system:admin"` is not needed here when using the expected kubeconfig.